specification/cognitiveservices/data-plane/SpeechServices/stable/v2.0/Speech.json

```diff
@@ -0,0 +1,13520 @@
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "The minimum number of seconds to wait before not getting this response anymore.",
+                "type": "integer"
+              }
+            },
+            "examples": {
+              "application/json": {}
+            }
+          }
+        },
+        "security": [
+          {
+            "subscription_key": [],
+            "token": []
+          }
+        ]
+      },
+      "patch": {
+        "tags": [
+          "Custom Speech datasets for model adaptation:"
+        ],
+        "summary": "Updates the mutable details of the dataset identified by its ID.",
+        "operationId": "UpdateDataset",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "description": "The identifier of the dataset.",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "name": "datasetUpdate",
+            "in": "body",
+            "description": "The updated values for the dataset.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DatasetUpdate"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Dataset"
+            },
+            "headers": {
+              "X-RateLimit-Limit": {
+                "description": "The limit of requests for the resource.",
+                "type": "integer"
+              },
+              "X-RateLimit-Remaining": {
+                "description": "The remaining number of requests until the rate limit is reached.",
+                "type": "integer"
+              },
+              "X-RateLimit-Reset": {
+                "description": "The timestamp when the rate limit will be reset formatted as ISO 8601 combined date and time in UTC.",
+                "type": "string"
+              }
+            },
+            "examples": {
+              "application/json": {
+                "dataImportKind": "Acoustic",
+                "id": "c0a8e65e-e61a-479f-8600-38dd2aaaf4ca",
+                "createdDateTime": "2019-03-20T14:46:18Z",
+                "lastActionDateTime": "2019-03-20T14:46:18Z",
+                "status": "Succeeded",
+                "locale": "en-US",
+                "name": "Acoustic dataset",
+                "properties": {
+                  "AcceptedLines": "11",
+                  "RejectedLines": "2",
+                  "ReportUri": "https://www.example.org/AcousticData/ed9b62f0-aa8e-419c-ab55-d7ca83c2f6b3?st=2018-02-09T18%3A07%3A00Z&se=2018-02-10T18%3A07%3A00Z&sp=r
```

Comment:
should this be tagged as long-running? - see https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-long-running-operation